### PR TITLE
Fix misspelled words

### PR DESCRIPTION
**Pull request by Madhu-1**
**Description of your changes:**
This  PR fixes misspelled words.

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**
- [ ] Documentation has been updated, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md#comments)

### CODE REVIEW

**Comment by Madhu-1** [comment]
@travisn PTAL

File: Documentation/edgefs-isgw-crd.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ metadata:
   name: isgw01                               # Inter-Segment Gateway service name
   namespace: rook-edgefs
 spec:
-  direction: "send+recieve"
+  direction: "send+receive"
```
Comment (Madhu-1) [inline Documentation/edgefs-isgw-crd.md]: @travisn this needs any code changes?

**Comment by galexrt** [comment]
@dyusupov requesting review from EdgeFS guys for their code and to answer @Madhu-1 questions.

File: Documentation/edgefs-isgw-crd.md (hunk 1 above)
Comment (dyusupov) [reply Documentation/edgefs-isgw-crd.md]: Looks good - thanks for catching this!

**Pull request merged**